projects/dxilconv/lib/DxilConvPasses/ScopeNestedCFG.cpp

```diff
@@ -393,6 +392,7 @@ bool ScopeNestedCFG::runOnFunction(Function &F) {
     BasicBlock *pHeader = pLoop->getHeader();
     LoopItem LI = m_LoopMap[pHeader];
     BasicBlock *pLatch = LI.pLL;
+    (void)(pLatch);
     DXASSERT_NOMSG(pLatch->getTerminator()->getNumSuccessors() == 1 && pLatch->getTerminator()->getSuccessor(0) == pHeader);
```

Comment:
It does occur to me only now that the (void)(exp) in DXASSERT_NOMSG means that these functions get called even on Release branches. I wonder if that's really what we want.